src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/events/DomainEvents.kt

```diff
@@ -12,7 +13,7 @@ abstract class DomainEvent<T : AdditionalInformation>(
   val eventType = eventType.eventType
   val description = eventType.description
   val version: String = "1"
-  val occurredAt: LocalDateTime = LocalDateTime.now()
+  val occurredAt: String = LocalDateTime.now().toOffsetString()
```

Comment:
Having this as a string is fine, we don't actually use it on receipt of events.  Having it as a local date time highlighted it was wrong anyway.